bin/mail/2016-04-26-changeorg-petition/send-ab-test-real-mails

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
For clarity, Is it possible to rename the "real" files without the word "test" in them? Perhaps `send-ab-experiment-{real,test}-mails`?